Formula/quantlib.rb

```diff
@@ -20,6 +20,7 @@ class Quantlib < Formula
   end
 
   option :cxx11
+  option "with-intraday", "Enable intraday components to dates"
```

Comment:
Enabling by default would not necessarily make sense as this was added later in QuantLib's life (v 1.7) and is still an "opt-in" feature. Primarily, some code may not have support for it in earlier quant software versions. Thus, it would be preferred to keep it as opted in and downstream can add documentation as to how to install (e.g.  https://github.com/eddelbuettel/rquantlib/wiki/RQuantLib )